Jellyfin.Api/Controllers/TvShowsController.cs

```diff
@@ -65,6 +65,7 @@ public class TvShowsController : BaseJellyfinApiController
         /// <param name="imageTypeLimit">Optional. The max number of images to return, per image type.</param>
         /// <param name="enableImageTypes">Optional. The image types to include in the output.</param>
         /// <param name="enableUserData">Optional. Include user data.</param>
+        /// <param name="nextUpDateCutoff">Starting date of shows to show in Next Up section.</param>
```

Comment:
It's still optional ;)
```suggestion
        /// <param name="nextUpDateCutoff">Optional. Starting date of shows to show in Next Up section.</param>
```

Reply:
ah doi. Silly mistake. Sorry about that. Fixed